android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java

```diff
@@ -432,7 +469,12 @@ public void onFinishScan(final ArrayList<ClientScanResult> clients) {
      * return {@code true} if Wi-Fi AP is enabled
      */
     private void isWiFiAPEnabled(Result poResult) {
-        poResult.success(moWiFiAPManager.isWifiApEnabled());
+        try {
```

Comment:
As far as I understand for API > 29 this would always fail at the moment. Would it make sense for API > 29 to return a value which we keep internally and update depending on `onStarted` / `onStopped` values we get as callbacks from `startLocalOnlyHotspot` ? (of course we could have also an `unknown` state if we never initialized it)

Reply:
As for now, I can't test on android 11 and above since I only have Android 10 and 5.

Reply:
created new issue #134 to track this, since it is stalling this PR.

Will add a note in README, so that we can know if this feature is required by user or not, if it is requested then we can think of implementing it.

Reply:
added note in b84c9d5c427c57750f6146546a69e984dd1e4082

Resolving this conversation as of now.